ci/deploy_app.yaml

```diff
@@ -2,37 +2,33 @@ platform: linux
 image_resource:
   type: docker-image
   source:
-    repository: ((image_registry))/eq-app-deploy-image
-    tag: ((deploy_image_version))
+    repository: gcr.io/google.com/cloudsdktool/cloud-sdk
+    tag: latest
```

Comment:
The ci Readme needs to be updated now that `image_registry` and `deploy_image_version` have been removed.

Reply:
Updated readme so it doesn't mention `image_registry` and `deploy_image_version`.